docs/interpreter/livy.md

```diff
@@ -61,6 +61,11 @@ Example: `spark.driver.memory` to `livy.spark.driver.memory`
     <td>Max number of Spark SQL result to display.</td>
   </tr>
   <tr>
+    <td>zeppelin.livy.spark.sql.truncate</td>
+    <td>true</td>
+    <td>Whether to truncate strings or not</td>
```

Comment:
right, to consider:
https://github.com/apache/zeppelin/blob/master/spark/src/main/java/org/apache/zeppelin/spark/SparkSqlInterpreter.java#L56